Configurations/Functions/Generic/ap-spd-vs-encoder-switcher.jinja2

```diff
@@ -1,7 +1,7 @@
 {% if data.get_simvar_value('AUTOPILOT_VERTICAL_HOLD') == 1.0 %}
-    {{ data.trigger_encoder_alternate(5, True) }}
-    {{ data.set_encoder_led(5, True) }}
+    {{ data.trigger_encoder_alternate(4, True) }}
```

Comment:
With the new feature of initializing variables from the config file we could move the encoder index to a variable. This makes sure this generic function is truly generic. Currently this change breaks the default config file
```jinja2
{% set encoder_index = get_global_variable('AUTOPILOT_ENCODER') %}
{% if data.get_simvar_value('AUTOPILOT_VERTICAL_HOLD') == 1.0 %}
    {{ data.trigger_encoder_alternate(encoder_index, True) }}
```

```json
"initialization": {
    "global_variables": [
      {
        "name": "AUTOPILOT_ENCODER",
        "value": 4
      }
    ]
  }
```

Reply:
Same of course applies for other functions that have been added to the generic folder

Reply:
> Currently this change breaks the default config file

Nevermind, just saw that the default config file was changed as well